portfolio/src/main/java/com/google/sps/servlets/InitServlet.java

```diff
@@ -21,7 +21,7 @@
 import com.googlecode.objectify.ObjectifyService;
 import com.google.sps.data.Comment;
 
-/** Servlet that performs initial operations. */
+/** Servlet that initializes the ObjectifyService. */
```

Comment:
nit: While this is fine, what I was thinking is more generic in that it says it initializes the datbase connection. So instead of mentioning what exact database interface you're using, just mention that it's being initialized.